resource-timing/resources/iframe-TAO-crossorigin-port.sub.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <script>
+let client = new XMLHttpRequest,
+    url = '{{location[scheme]}}://{{host]}}:{{ports[http][1]}}/resource-timing/resources/TAOResponse.py?tao=origin';
+    client.open('GET', url);
+    client.onload =(()=>{
+        setTimeout(()=>{
+            let entry = performance.getEntriesByType('resource')[0];
+            let result = (entry && (entry.redirectStart + entry.redirectEnd + entry.domainLookupStart + entry.domainLookupEnd + entry.connectStart + entry.connectEnd + entry.secureConnectionStart + entry.requestStart + entry.responseStart) == 0) ? 'PASS' : 'FAIL';
```

Comment:
Pass here is saying that the sum is 0, doesn't that mean that the TAO check failed?

Reply:
Yeah, it was testing the wrong thing. Added another "origin_port" TAOResponse destination which returns a different port and should fail